app/models/grda_warehouse/file.rb

```diff
@@ -7,6 +7,7 @@
 module GrdaWarehouse
   class File < GrdaWarehouseBase
     acts_as_paranoid
+    has_paper_trail
```

Comment:
This caused a test regression on HMIS, because duplicate versions are being created for HMIS files. https://github.com/greenriver/hmis-warehouse/actions/runs/11525170831/job/32086897713

The problem is that `Hmis::File` [also calls](https://github.com/greenriver/hmis-warehouse/blob/6ec65c770a43d29291ecbed7918979fa27e5384a/drivers/hmis/app/models/hmis/file.rb#L10-L16) `has_paper_trail` (and attaches some metadata). 

Paper Trail has an unreleased (breaking) change that will actually cause this to fail https://github.com/paper-trail-gem/paper_trail/blob/master/CHANGELOG.md#breaking-changes / https://github.com/paper-trail-gem/paper_trail/pull/1479. They acknowledge that this can create duplicates. Glad we caught it with the test first!

I don't have much content on this PR, so not sure what the best course of action is. This class is pretty lightweight, so one approach is for Hmis::File to just not subclass it. I did that here https://github.com/greenriver/hmis-warehouse/pull/4872 and the tests pass, but it needed some weirdness to get `type` set.

Reply:
Good catch @gigxz !  I believe our intent was actually to add `has_paper_trail` to ClientFile.  Not sure we need it in the base class.  I'll give that a try and see if tests still fail.